packages/react-image/src/components/Image/Image.tsx

```diff
@@ -1,10 +1,22 @@
-import { compose, createClassResolver } from '@fluentui/react-compose';
-
-import { ImageBase } from './ImageBase';
+import * as React from 'react';
 import { ImageProps } from './Image.types';
+import { useFocusRects } from '@uifabric/utilities';
+import { makeClasses } from '@fluentui/react-compose/lib/next/index';
+import { useInlineTokens } from '@fluentui/react-theme-provider';
 import * as classes from './Image.scss';
+import { useImage } from './useImage';
+
+// Create a hook to resolve classnames.
+export const useImageClasses = makeClasses(classes);
+
+export const Image = React.forwardRef<HTMLElement, ImageProps>((props, ref) => {
+  const { render, state } = useImage(props, ref);
+
+  // Apply styling.
+  useImageClasses(state);
+  useFocusRects(state.imageRef);
```

Comment:
```suggestion
```

I don't think that `Image` should have focus behavior. In the spec it's also not defined